content/en/build-guide/_index.adoc

```diff
@@ -1,31 +1,33 @@
 
 ---
-title: "PAPRa Build Guide (Jan 31 2021 Edition)"
+title: "PAPRa Build Guide (March 2021 Edition)"
 linkTitle: "PAPRa Build Guide"
 weight: 1
 description: >
-  How to build the January 31 2021 version of the Tetra PAPR.
+  How to build the March 2021 version of the Tetra PAPR.
 ---
 
 == Quick Synopsis
 
 This guide will walk you through how to build a Personal Air Purifying Respirator, also known as a PAPR.  We assume that you have some knowledge of 3D-printing, and cutting foam with a knife or using a stencil or a cricut, and perhaps some soldering.  In the end, you will have four things: a box with a filter and a fan in it, a second box with batteries and a fan controller, a mask, and straps and hoses to hold it all together. 
 
-Please note that this system will not protect you from anything landing in your eyes.  For that protection, either the mask will need to be modified to be a helmet, or you will need a face shield.
+Please note that this system will not protect you from anything landing in your eyes.  For that protection, either the mask will need to be modified to be a helmet or you will need a face shield.
+
+An important point: we provide no warranty, implicit or explicit, that this device will work up to an N95 specification, when built outside of a facility rated to do so.  The only way to truly test one of these devices using something like a https://tsi.com/products/respirator-fit-testers/portacount-respirator-fit-tester-8038/[Portacount] under controlled conditions.
```

Comment:
Personally, I would convert this text block to an [admonition](https://docs.asciidoctor.org/asciidoc/latest/blocks/admonitions/), replacing _An important point_ with **WARNING**, but it's purely an aesthetic choice.